examples/rsp-next-ts/package.json

```diff
@@ -17,15 +17,15 @@
     "@react-spectrum/toast": "^3.0.0-alpha.0",
     "@spectrum-icons/illustrations": "^3.5.0",
     "@spectrum-icons/workflow": "^4.0.3",
-    "next": "^13.1.1",
+    "next": "13.2.4",
```

Comment:
Seems like its been fixed in `next@13.3.2-canary.6`, preferred a stable version here instead. Perfectly fine with going with the canary release if people think that is better